pkg/cmd/serviceaccount/create/create.go

```diff
@@ -94,7 +94,7 @@ func NewCreateCommand(f *factory.Factory) *cobra.Command {
 	cmd.Flags().StringVar(&opts.name, "name", "", opts.localizer.MustLocalize("serviceAccount.create.flag.name.description"))
 	cmd.Flags().StringVar(&opts.description, "description", "", opts.localizer.MustLocalize("serviceAccount.create.flag.description.description"))
 	cmd.Flags().BoolVar(&opts.overwrite, "overwrite", false, opts.localizer.MustLocalize("serviceAccount.common.flag.overwrite.description"))
-	cmd.Flags().StringVar(&opts.filename, "file-location", "", opts.localizer.MustLocalize("serviceAccount.common.flag.fileLocation.description"))
+	cmd.Flags().StringVar(&opts.filename, "output-file", "", opts.localizer.MustLocalize("serviceAccount.common.flag.fileLocation.description"))
```

Comment:
In this case we dealing with input file.
I'm not 100% convinced we need output/input as that can be derrived from command:

- get downloads file
- create sends file

It is less typing

Reply:
Anyway this is and edit command should have different values than rest.
Apart from that all good

Reply:
No, we are dealing with an output file here. Yo create a service account and it outputs the credentials to a file.

Reply:
> Anyway this is and edit command should have different values than rest.

Sorry, not sure I understand here?

Reply:
OMG - service account not service registry :).

SHAME! Let's forget about this 📸